test/api/support/validateAgainstJSONSchema.js

```diff
@@ -1,25 +1,27 @@
-const deref = require('json-schema-deref-sync')
-const validate = require('jsonschema').validate
-const YAML = require("yamljs")
+const deref = require('json-schema-deref-sync');
+const validate = require('jsonschema').validate;
+const YAML = require('yamljs');
 
-const swagger = YAML.load("public/swagger/index.yml")
+const swagger = YAML.load('public/swagger/index.yml');
 const schema = deref(swagger, {
-  baseFolder: process.cwd() + "/public/swagger",
+  baseFolder: `${process.cwd()}/public/swagger`,
   failOnMissing: true,
-})
+});
 
 const validateAgainstJSONSchema = (action, path, statusCode, response) => {
-  action = action.toLowerCase()
-  path = path.toLowerCase()
-  statusCode = parseInt(statusCode)
+  const actionLower = action.toLowerCase();
+  const pathLower = path.toLowerCase();
+  const statusCodeInt = parseInt(statusCode, 10);
 
-  const responseSchema = schema.paths[path][action].responses[statusCode].schema
-  const result = validate(response,responseSchema)
+  const responseSchema = schema.paths[pathLower][actionLower].responses[statusCodeInt].schema;
+  const result = validate(response, responseSchema);
 
   if (result.errors.length) {
-    console.error(result.errors)
-    throw new Error(`Failed to validate against definition: ${definition}`)
```

Comment:
Fixed a minor bug here -- `definition` is not defined anywhere so the error message wasn't very helpful. Since I fixed that bug, I also had to lint the file.